bin/run.sh

```diff
@@ -70,7 +70,7 @@ export JAVA_TOOL_OPTIONS="-Djansi.tmpdir=/solution/jansi-tmp -Xss128m -Xms256m -
 rm -rf target
 # Run the tests for the provided implementation file and redirect stdout and
 # stderr to capture it
-test_output=$(mvn --offline --legacy-local-repository --batch-mode --non-recursive --quiet test 2>&1)
```

Comment:
This flag has been dropped since 3.9.1: https://issues.apache.org/jira/browse/MNG-7713
It works without it in Docker locally, was it something specific to running on the cloud?